demo/blocks_joined/run.py

```diff
@@ -26,25 +26,17 @@ def img(text):
         "DALL·E mini is an AI model that generates images from any prompt you give!"
     )
     with gr.Group():
-        with gr.Box():
-            with gr.Row().style(equal_height=True):
-
-                text = gr.Textbox(
-                    label="Enter your prompt", show_label=False, max_lines=1
-                ).style(
-                    border=(True, False, True, True),
-                    rounded=(True, False, False, True),
-                    container=False,
-                )
-                btn = gr.Button("Run").style(
-                    margin=False,
-                    rounded=(False, True, True, False),
-                )
-        gallery = gr.Gallery(label="Generated images", show_label=False).style(
-            grid=(
-                1,
-                3,
-            ),
+        with gr.Row(equal_height=True):
+            text = gr.Textbox(
+                label="Enter your prompt",
+                max_lines=1,
+                container=False,
+            )
+            btn = gr.Button("Run", scale=0)
+        gallery = gr.Gallery(
+            label="Generated images",
+            show_label=False,
+            grid=(1, 3),
```

Comment:
this parameter doesn't exist on `gr.Gallery` and causes this warning

`GradioUnusedKwargWarning: You have unused kwarg parameters in Gallery, please remove them: {'grid': (1, 3)}`